roles/orangelight/vars/main.yml

```diff
@@ -7,5 +7,4 @@ rails_app_dependencies:
   - pkg-config
   - libtool
   - autoconf
-passenger_ruby: "/usr/bin/ruby2.7"
-ruby_version_override: "ruby2.7"
+
```

Comment:
`molecule lint` is complaining about this extra line.  Could you please remove it?

Reply:
yes I will.